mlrun/model_monitoring/db/v3io_tsdb_reader.py

```diff
@@ -132,3 +133,91 @@ def read_data(
+    endpoint_id: str,
+    start: Optional[datetime] = None,
+    end: Optional[datetime] = None,
+    aggregation_window: Optional[str] = None,
+    limit: Optional[int] = None,
+) -> Union[
+    mlrun.common.schemas.model_monitoring.model_endpoints.ModelEndpointMonitoringResultValues,
+    mlrun.common.schemas.model_monitoring.model_endpoints.ModelEndpointMonitoringResultNoData,
+]:
+    client = mlrun.utils.v3io_clients.get_frames_client(
+        address=mlrun.mlconf.v3io_framesd,
+        container="users",
+    )
+    frames_client_kwargs = {}
+    if aggregation_window:
+        frames_client_kwargs["step"] = aggregation_window
+        frames_client_kwargs["aggregation_window"] = aggregation_window
+    if limit:
+        frames_client_kwargs["limit"] = limit
+    df: pd.DataFrame = client.read(
+        backend=_TSDB_BE,
+        table=f"pipelines/{project}/model-endpoints/predictions",
+        columns="latency",
+        filter=f"endpoint_id=='{endpoint_id}'",
+        start=start,
+        end=end,
+        aggregators="count",
+        **frames_client_kwargs,
+    )
+
+    full_name = (
+        mlrun.common.schemas.model_monitoring.model_endpoints._compose_full_name(
+            project=project,
+            app="mlrun-infra",
+            name=mlrun.common.schemas.model_monitoring.MonitoringTSDBTables.INVOCATIONS,
+        )
+    )
+
+    if df.empty:
+        return ModelEndpointMonitoringResultNoData(
+            full_name=full_name,
+            type=ModelEndpointMonitoringMetricType.METRIC,
+        )
+
+    rows = df.reset_index().to_dict(orient="records")
+    values = [
+        [
+            row["time"],
+            row["count(latency)"],  # event count for the time window
+            mm_constants.ResultStatusApp.irrelevant,
+        ]
+        for row in rows
+    ]
+    return ModelEndpointMonitoringResultValues(
+        full_name=full_name,
+        type=ModelEndpointMonitoringMetricType.RESULT,
+        result_kind=mm_constants.ResultKindApp.system_performance,
+        values=values,
+    )
+
+
+def read_prediction_metric_for_endpoint_if_exists(
+    *,
+    project: str,
+    endpoint_id: str,
+) -> Optional[ModelEndpointMonitoringMetric]:
+    predictions = read_predictions(
+        project=project,
+        endpoint_id=endpoint_id,
+        limit=1,  # Read just one record, because we just want to check if there is any data for this endpoint_id
+    )
+    if predictions:
+        return ModelEndpointMonitoringMetric(
+            project=project,
+            app="mlrun-infra",
+            type=ModelEndpointMonitoringMetricType.METRIC,
+            name=mlrun.common.schemas.model_monitoring.MonitoringTSDBTables.INVOCATIONS,
+            full_name=mlrun.common.schemas.model_monitoring.model_endpoints._compose_full_name(
+                project=project,
+                app="mlrun-infra",
+                name=mlrun.common.schemas.model_monitoring.MonitoringTSDBTables.INVOCATIONS,
+            ),
```

Comment:
Here you have the FQN, but it's also somewhat wrong:
1. It should be a metric (through the `type` parameter). Ok to keep for now, I'll change it later.
2. The app should be saved as a constant, see the previous comment.